sdks/python/apache_beam/io/jdbc.py

```diff
@@ -128,11 +128,11 @@ def default_io_expansion_service(classpath=None):
      ('disable_autocommit', typing.Optional[bool]),
      ('output_parallelization', typing.Optional[bool]),
      ('autosharding', typing.Optional[bool]),
+     ('write_batch_size', typing.Optional[np.int64]),
      ('partition_column', typing.Optional[str]),
      ('partitions', typing.Optional[np.int16]),
      ('max_connections', typing.Optional[np.int16]),
-     ('driver_jars', typing.Optional[str]),
-     ('write_batch_size', typing.Optional[np.int64])],
```

Comment:
Can we move the ordering in the Java SchemaTransform instead? 
I'm thinking some users may be passing args to WriteToJdbc() relying on ordering. Putting write_batch_size in the middle may break them.